analytic_base_department/__manifest__.py

```diff
@@ -3,13 +3,12 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Base Analytic Department Categorization",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "author": "Camptocamp, Daniel Reis, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "category": "Generic Modules/Projects & Services",
     "website": "https://github.com/OCA/account-analytic",
     "depends": ["account",
-                "analytic",
```

Comment:
Why is this removed? The module uses `account.analytic.account` Model, provided by the `analytic` module.

Reply:
@dreispt ,

If we will install the `account` module then `analytic` module will automatically installed due to dependency. So no need to put `analytic` module in the dependency
Here is the Reference:- https://github.com/odoo/odoo/blob/12.0/addons/account/__manifest__.py#L18